test/new-e2e/utils/infra/stack_manager.go

```diff
@@ -141,6 +142,17 @@ func (sm *StackManager) DeleteStack(ctx context.Context, name string) error {
 	return sm.deleteStack(ctx, name, stack)
 }
 
+func (sm *StackManager) ForceRemoveStack(ctx context.Context, name string) error {
+	sm.lock.Lock()
+	defer sm.lock.Unlock()
+
+	stack := sm.stacks[name]
```

Comment:
does this always succeed?

Reply:
Yes it should always succeed.

Reply:
In the current usage, it should always succeed but in the general case it may fail if the stack was not created.